### PR TITLE
Using Branch for django-announcements

### PR DESCRIPTION
**Pull request by clintonb**
Using the branch since I accidentally rebased and destroyed the commit.

@rocha @dsjen

### CODE REVIEW

**Comment by dsjen** [comment]
Looks good to me

File: requirements/base.txt [hunk 1]
```diff
@@ -13,7 +13,8 @@ django-waffle==0.10			# BSD
 # django-countries==3.0.1     # MIT
 git+https://github.com/SmileyChris/django-countries.git@ece149d706aaf812afd6b39872cf01fd294b2038#egg=django-countries
 
--e git+https://github.com/clintonb/django-announcements.git@e5c4ef825111befdc2dd2b206fbb322367ae5a3e#egg=django-announcements # MIT
+# TODO Use the edX fork, once it is created.
+-e git+https://github.com/clintonb/django-announcements.git@django-1.7#egg=django-announcements # MIT
```
Comment (rocha) [inline requirements/base.txt]: Can we wait for the edX fork before merging?

Comment (clintonb) [inline requirements/base.txt]: Yeah. I was thinking it might take a while; but, I believe @maxrothman is working on the fork now.

**Comment by clintonb** [comment]
Now using the edX fork. Will update to the PyPi package once it is released; but, we shouldn't hold our breaths since it took nearly two months to get the PR merged.

**Pull request merged**